homeassistant/components/mastodon/quality_scale.yaml

```diff
@@ -0,0 +1,102 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: |
+      Legacy Notify platform can't be created within async-setup.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow:
+    status: done
+    comment: |
+      YAML flow deprecated and will be removed in 2025.2.0
```

Comment:
I want to try to avoid having hard numbers or promises in this file

Reply:
OK, I'll just mark as done, the yaml flow is migrated via import flow.